src/functions/node.php

```diff
@@ -21,3 +25,51 @@ function node_view($node, $view_mode = 'full', $langcode = null)
       ->getViewBuilder('node')
       ->view($node, $view_mode, $langcode);
 }
+
+function node_type_get_name(EntityInterface|string $node): string|false
+{
+    if ($node instanceof EntityInterface) {
+        return $node->bundle();
+    } else {
+        $types = NodeType::loadMultiple();
+        return isset($types[$node]) ? $node : false;
```

Comment:
I was going to say we can be fancy and use PHP 8.1 language features:

```php
return \Drupal\node\Entity\NodeType::load($node)?->label() ?? false;
```

But I don't know if that is better or not. We could at least use `load` over `loadMultiple`.